CHANGELOG.md

```diff
@@ -24,8 +24,7 @@ import { SimpleCache } from 'fastly:cache';
 addEventListener('fetch', event => event.respondWith(app(event)));
 
 async function app(event) {
-  const url = new URL(event.request);
-  const path = url.pathname;
+  const path = new URL(event.request.url).pathname;
```

Comment:
we need the `url` variable for the if statement in the line below

Reply:
Oops, sorry. How's https://github.com/fastly/js-compute-runtime/pull/580/commits/7d64b3d8ab657585cea29a83c7e56c8bceda9527 ?